### DISCUSSION TITLE
SEO Plugin types

### DISCUSSION THREAD

**Discussion by jstask82** [payloadcms/payload#7618] [Q&A]
i am on payload 2.26.0 and i did interagte the [SEO Plugin](https://github.com/payloadcms/payload/tree/main/packages/plugin-seo) as [described in the docs](https://payloadcms.com/docs/plugins/seo), however the docs in ```generateTitle: ({ doc }) => `Website.com — ${doc.title.value}`, generateDescription: ({ doc }) => doc.excerpt``` is `any`. Are there types available I can use?

**Comment by jstask82** [comment]
```
generateTitle: ({ doc }) => {
  const possibleDocs = doc as MyCollection | MyOtherCollection;
  return `Website.com – ${possibleDocs.title}`;
}
```
A type assertion can be used, to get rid of the any type

**Reply by andershermansen** [reply]
This is the way to go for version 2. They fixed it better for version 3. See the breaking changes for beta 56: https://github.com/payloadcms/payload/releases/tag/v3.0.0-beta.56
